internal/storage/engine.go

```diff
@@ -53,21 +53,31 @@ func buildIssuerFromSeed(seed SeedIssuer) (types.Issuer, error) {
 		return types.Issuer{}, err
 	}
 
+	claimConditions, err := types.NewClaimConditions(seed.ClaimConditions)
+	if err != nil {
+		return types.Issuer{}, err
+	}
+
 	out := types.Issuer{
-		OwnerID:       seed.OwnerID,
-		ID:            seed.ID,
-		Name:          seed.Name,
-		URI:           seed.URI,
-		JWKSURI:       seed.JWKSURI,
-		ClaimMappings: claimMappings,
+		OwnerID:         seed.OwnerID,
+		ID:              seed.ID,
+		Name:            seed.Name,
+		URI:             seed.URI,
+		JWKSURI:         seed.JWKSURI,
+		ClaimMappings:   claimMappings,
+		ClaimConditions: claimConditions,
 	}
 
 	return out, nil
 }
 
+// TestServerCRDBVersion is the version of CockroachDB that the test server is running
+// v23.1.28 is the last version under BSL
+const TestServerCRDBVersion = "v23.1.28"
```

Comment:
Another solution to this is to set the crdb binary if it already exists.

https://github.com/infratographer/permissions-api/blob/main/.devcontainer/Dockerfile#L21-L25